server/controllers/RequestsController.js

```diff
@@ -0,0 +1,30 @@
+import Controller from './Controller';
+import requests from '../dummy-data/requests';
+
+class RequestController extends Controller {
+  /**
+   * @description Fetch all the requests of a logged in user
+   *
+   * @param {Object} req - HTTP Request
+   * @param {Object} res - HTTP Response
+   *
+   * @return {Object} Returned object
+   */
+  static getRequests(req, res) {
+    const userId = parseInt(req.headers['user-id'], 10);
```

Comment:
Try to handle more errors and edge cases, what happens when the 'user-id' is a string?